react/ProductLink.tsx

```diff
@@ -68,8 +68,19 @@ function ProductLink(props: Props) {
     [classes.label]: displayMode === 'button',
   })
 
+  const handlePrevent = (e: React.MouseEvent) => {
+    e.stopPropagation()
+    e.nativeEvent.stopImmediatePropagation()
+  }
+
   return (
-    <Link target={target} to={resolvedLink} className={rootClasses} rel={rel}>
+    <Link
+    target={target}
+    to={resolvedLink}
+    className={rootClasses}
+    rel={rel}
+    onClick={handlePrevent}
+    >
```

Comment:
You forgot to prettify.

```suggestion
    <Link
      target={target}
      to={resolvedLink}
      className={rootClasses}
      rel={rel}
      onClick={handlePrevent}
    >
```